### PR TITLE
first pass at doc

### PR DESCRIPTION
**Pull request by apfister**
just a first pass at adding some documentation to the notebook.

### CODE REVIEW

File: PublishItemCards.ipynb [hunk 1]
```diff
@@ -4,40 +4,77 @@
...
+     "output_type": "stream",
+     "text": [
+      "Logged in as: apfister_sdg\n"
+     ]
+    }
+   ],
    "source": [
-    "online_username = \"\"\n",
-    "online_password = \"\"\n",
-    "online_connection = \"http://undesa.maps.arcgis.com\"\n",
```
Comment (apfister) [inline PublishItemCards.ipynb:38]: I think we can just use `https://www.arcgis.com` here, no?

**Review by travisbutcher — APPROVED** [review]
Dropping the UN org makes it more generic

**Comment by apfister** [comment]
hmm.. does this mean we also need to update any code changes we do in the notebook to https://github.com/UNStats-SDGs/sdg-publisher/blob/master/PublishItemCards.py as well?

**Comment by travisbutcher** [comment]
yes.  I will handle that, any updates in the notebook I have moving into the standard py file as well

**Pull request merged**